driver/pace/driver/run.py

```diff
@@ -116,85 +130,172 @@ class SerialboxConfig(InitializationConfig):
     """
 
     path: str
-    npx: int
-    npy: int
-    npz: int
-    layout: List[int]
-    backend: str
+    serialized_grid: bool
 
     @property
     def start_time(self) -> datetime:
         return datetime(2000, 1, 1)
 
+    @property
+    def f90_namelist(self) -> f90nml.Namelist:
+        return f90nml.read(self.path + "/input.nml")
+
+    @property
+    def namelist(self) -> Namelist:
+        return Namelist.from_f90nml(self.f90_namelist)
```

Comment:
Can these be private attributes?

Reply:
Yes, forgot to update this now that we don't need to call them from outside the class.